koku/masu/database/sql/reporting_azurecostentrylineitem_daily_summary.sql

```diff
@@ -1,29 +1,56 @@
 -- Place our query in a temporary table
 CREATE TEMPORARY TABLE reporting_azurecostentrylineitem_daily_summary_{{uuid | sqlsafe}} AS (
+    WITH cte_split_units AS (
+        SELECT li.id,
+            m.currency,
+            CASE WHEN split_part(m.unit_of_measure, ' ', 2) != '' AND m.unit_of_measure != '100 Hours'
```

Comment:
```suggestion
            CASE WHEN split_part(m.unit_of_measure, ' ', 2) != '' AND NOT (m.unit_of_measure = '100 Hours AND m.meter_category='Virtual Machines')'
```

Feel free to ignore my use of NOT replacing the != you had, but I do think adding in the meter_category is good for now since we only know for sure that the virtual machine data is impacted